binding.cc

```diff
@@ -1179,6 +1180,26 @@ namespace zmq {
     NanReturnValue(NanNew<String>(version_info));
   }
 
+#if ZMQ_VERSION_MAJOR >= 4
+   static NAN_METHOD(ZmqCurveKeypair) {
+    NanScope();
+
+    char public_key [41];
+    char secret_key [41];
+
+    int rc = zmq_curve_keypair( public_key, secret_key);
+    if (rc < 0) {
+      throw std::runtime_error(ErrorMessage());
```

Comment:
I also found now that on line 969/970 in the ```Socket::Unmonitor``` NanMethod also an ```std::runtime_error``` is thrown. It is out of the scope of this change but seems very similar to this situation so that one may need to be changed as well? (Or am I completely wrong?)